ffi/Cargo.toml

```diff
@@ -5,7 +5,16 @@ authors = ["Iroha 2 team <https://github.com/orgs/soramitsu/teams/iroha2>"]
 edition = "2021"
 
 [features]
-wasm = []
+# TODO:
+# Whether or not to perform validity checks when converting from C type to Rust type.
+# This is useful when the side you are receiving data from can be considered trusted.
+# Most notably this feature can be used in smart contracts that interact with `Iroha`
+# no_checks = []
```

Comment:
The assumption here was that `Iroha` can be trusted to be without bugs and will always send valid values. For instance, it will never send `3` when `core::cmp::Ordering` is expected (valid values are `-1`, `0`, `1`) or `null` for a pointer, etc. Given this assumption smart contract can shave of a few bytes by not including code for doing these checks

Reply:
I'll open a ticket for this and remove the comment

Reply:
#2799